src/main/java/core/basesyntax/service/cvs/FileWriterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.cvs;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileWriterImpl implements FileWriter {
+    private CsvReportService csvReportService;
+
+    public FileWriterImpl(CsvReportService csvReportService) {
+        this.csvReportService = csvReportService;
+    }
+
+    @Override
+    public void writeToFile(String toFile) {
```

Comment:
It is better not to use such a binding, but to pass the data that we want to write to this method.
The task of FileWriter is to write data. But what if we want to record the data received not from the report, but from another place? Thus it is necessary to write one more class.
Also, such code will be difficult to test.